### PR TITLE
Create a hashing utility

### PR DESCRIPTION
**Pull request by mborof**

### CODE REVIEW

File: src/utils/string.js [hunk 1]
```diff
@@ -165,3 +166,37 @@ export function getUuid() {
...
+  for (let i = 0; i < view.byteLength; i += 4) {
+    // toString(16) will give the hex representation of the number without padding
+    const stringValue = view.getUint32(i).toString(16);
+    hexCodes.push(padString(stringValue, PADDING));
+  }
+  return hexCodes.join('');
+}
+
+/**
+ * @param {string} stringToHash
```
Comment (ChrisAntaki) [inline src/utils/string.js:193]: Please add a comment for this method

File: src/utils/string.js [hunk 2]
```diff
@@ -165,3 +166,37 @@ export function getUuid() {
 export function getSwgTransactionId() {
   return getUuid() + '.swg';
 }
+
+/**
+ * Returns a string who's length matches the length of format.
```
Comment (ChrisAntaki) [inline src/utils/string.js:171]: Please update "who's" to "whose"

File: src/utils/string-test.js [hunk 3]
```diff
@@ -162,3 +163,19 @@ describe('swgTransactionId', () => {
     }
   });
 });
+
+describe('hash', async () => {
+  it('should create 128 character hexadecimal hashes', async () => {
+    let result = null;
```
Comment (ChrisAntaki) [inline src/utils/string-test.js:169]: Please combine this line and the line below it. Ex: `const result = await ...`

File: src/utils/string-test.js [hunk 4]
```diff
@@ -162,3 +163,19 @@ describe('swgTransactionId', () => {
...
+    await hash('string1').then((h) => (result = h));
+    expect(result.length).to.equal(128);
+  });
+
+  it('should create unique hashes', async () => {
+    let hash1 = null;
+    let hash2 = null;
```
Comment (ChrisAntaki) [inline src/utils/string-test.js:176]: Please use the values of the `await` expressions to define these variables more succinctly. Ex: `const hash1 = await ...`

File: src/utils/string-test.js [hunk 5]
```diff
@@ -162,3 +163,19 @@ describe('swgTransactionId', () => {
     }
   });
 });
+
+describe('hash', async () => {
+  it('should create 128 character hexadecimal hashes', async () => {
+    let result = null;
+    await hash('string1').then((h) => (result = h));
+    expect(result.length).to.equal(128);
```
Comment (ChrisAntaki) [inline src/utils/string-test.js:171]: Please check against an actual hash value instead of length

File: src/utils/string.js (hunk 1 above)
Comment (mborof) [reply src/utils/string.js:193]: To be honest, I'm not sure what a good comment is for this function.

Comment (ChrisAntaki) [reply src/utils/string.js:193]: Could you describe what it does, and what if might be used for?

File: src/utils/string-test.js (hunk 5 above)
Comment (mborof) [reply src/utils/string-test.js]: I'm not sure I agree in this case, we can discuss over chat if you'd like.  I did improve the tests some.

**Pull request merged**